testing/regress/ecl/modelingWithR.ecl

```diff
@@ -87,7 +87,15 @@ resultRec runAnalyses(DATASET(irisRec) ds) := EMBED(R)
     capture.output(summary(glmMdl))
   )
   fit<-data.frame(dsFit, stringsAsFactors = F)
-  
+
+  # This code generates different results after the 13th decimal digit
+  # on Ubuntu 16.04 and CentOS
+  # To using envronment independent key file to check the result of this test,
```

Comment:
Should say "To be able to use" rather than "To using" - and there's an "i" missing in environment